README.md

```diff
@@ -1,11 +1,36 @@
+
 # cachedcomplete
+
 Fast semi-static autocomplete for python
 
-## Normal Usage
-The normal usage is the same as in argcomplete, only changing the import from argcomplete to cachedcomplete
+## Installation
+
+Use the package manager pip to install cachedcomplete.
+
+```bash
+pip install cachedcomplete
+```
+
+## Explenation
```

Comment:
*Explanation
(might want to install a spell checker)

Reply:
I personally recommend "Grammarly"
It's really cool and very helpful and integrates with almost everything.

Reply:
> *Explanation
> (might want to install a spell checker)

Test

Reply:
> I personally recommend "Grammarly"
> It's really cool and very helpful and integrates with almost everything.

They're not so good when using GitHub or editing code.
I meant he should install a VSCode plugin.